### DISCUSSION TITLE
Certificate not issued on dev.opencerts.io

### DISCUSSION THREAD

**Discussion by ygengygeng** [OpenCerts/opencerts-website#664] [Q&A] [answered]
Hi

I've gotten user to issue on blockchain but dev cert is showing 'Certificate not issued' on dev.opencerts.io

It's issued to cert store 0xe825e69383a57eB686Fc7b8c455Dd982bB9680f6

https://etherscan.io/tx/0xafe2d75518ffba386020d2d733bfdaafdaf60215ba63dc2e55e76adcb9afad2c

Could you help me to check what could be the reason? Thanks.

**Comment by ygengygeng** [comment]
Sorry, is anyone available to help me check?

**Comment by cavacado** [comment, marked as answer]
hi, are you mixing certain things up?
cert store 0xe825e69383a57eB686Fc7b8c455Dd982bB9680f6 is a DocumentStore on the ropsten network
h/ever the transaction hash you provided, is a transaction made on the mainnet.
upon looking at previous transactions from this particular wallet 0xbe8ace22d3a8340c906d549087489fc3ac9aae5f (i assume u have control of); it seems that you used to issue from the DocumentStore 0x4D05F843718dA742b4229938f972a3EB969C25CE

whats your intention? issue a cert on the ropsten network or the cert on the mainnet?

**Reply by ygengygeng** [reply]
Hi

It's to issue a cert on the ropsten network.

Let me check with the wallet acccount holder. Thank you.
